spec/factories/events.rb

```diff
@@ -123,6 +123,12 @@
       end
     end
 
+    trait :with_enough_free_places do
+      after(:build) do |event|
+        event.max_participants = event.application_letters.size
```

Comment:
I'm for setting it by hand or making a trait like "adds application and enough free spaces" (because that would imply that it might not be wise to use it with other application traits, like in draft phase and in execution phase would not make sense together)